.github/workflows/knative-boilerplate.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
           go get github.com/mattmoor/boilerplate-check/cmd/boilerplate-check
           echo '::endgroup::'
 
-          echo "::add-path::${TEMP_PATH}"
+          echo "${TEMP_PATH}" >> $GITHUB_PATH
```

Comment:
Is that a leftover from some automated update?

Reply:
Correct it fixes the github action failures.